packages/lib/src/components/charts/lume-bar-chart/lume-bar-chart.spec.ts

```diff
@@ -9,7 +9,8 @@ import {
   yScale,
 } from '@test/unit/mock-data';
 import { flushPromises } from '@test/unit/flush-promises';
-import { CHART_TYPES, ChartType } from '@/utils/constants';
+import { CHART_TYPES } from '@/utils/constants';
+import type { ChartType } from '@/types/utils';
```

Comment:
We should probably consolidate these types/enums. For example, I don't think we need `CHART_TYPES` honestly - only being used in test files

Reply:
Updated ;)